rust_src/src/symbols.rs

```diff
@@ -187,6 +192,18 @@ impl LispSymbolRef {
         LispSymbolIter { current: self }
     }
 
+    pub fn get_next(self) -> Option<LispSymbolRef> {
```

Comment:
You PR comment above about why `nth` is used would make a good code comment here.

Reply:
Done.